src/tests/functional/plugin/conformance/test_runner/conformance_infra/src/skip_tests_config.cpp

```diff
@@ -18,7 +18,12 @@ const char *targetPluginName = "";
 const char *refCachePath = "";
 
 std::vector<std::string> IRFolderPaths = {};
-std::vector<std::string> disabledTests = {};
+std::vector<std::string> disabledTests = {
+
+    // CPU and GPU do not support dynamic rank
+    // Issue: 66778 and 98723
+    R"(.*OVInferRequestDynamicTests.*/*_targetDevice=(AUTO).*)",
+};
```

Comment:
how about other dynamic cases not with dynamic rank? we should not disable all of them?

Reply:
Update to skip the case of the dynamic rank in this test suit.

Reply:
The conformance concept is not to use skips at all. We demonstrate real counters without hacks